contracts/rewards/src/contract/execute.rs

```diff
@@ -30,6 +28,9 @@ where
         let stored_params = self.store.load_params();
         let epoch_duration: u64 = stored_params.params.epoch_duration.into();
         let epoch = stored_params.last_updated;
+        if cur_block_height < epoch.block_height_started {
+            return Err(ContractError::BlockHeightPassed.into());
+        }
         if cur_block_height >= epoch.block_height_started + epoch_duration {
```

Comment:
```suggestion
        if cur_block_height < epoch.block_height_started + epoch_duration {
          return Ok(epoch)
        }

        ...
```
I always code to look cleaner when exit early and have the complicated code reside in where the least amount of indentations are.